src/IdentityManager/WebApi/appsettings.Development.json

```diff
@@ -7,6 +7,6 @@
     }
   },
   "ConnectionStrings": {
-    "IdentityServerDB": "Server=DESKTOP-P5KNT2D\\MSSQLSERVER01;Database=IdentityServerDB;Trusted_Connection=True;" // my local conn string
+    "IdentityServerDB": "Server=DESKTOP-HBRRGTD;Database=IdentityServerDB;Trusted_Connection=True;" // my local conn string
```

Comment:
Please change to following connectionString
`Data Source=(LocalDb)\MSSQLLocalDB;Initial Catalog=TravelyIdentityService;Integrated Security=SSPI;AttachDBFilename=|DataDirectory|\TravelyIdentityService.mdf`

For local development use `appsettings.local.json` (you might require to create it manually). It is git-ignored and will not be pushed to git. So we will not have conflicts and each developer can use his own config.